test/baselines/generated-reference/namespaces.html

```diff
@@ -63,10 +65,14 @@ <h1><span class="secnum">A</span> Annex</h1>
     </emu-rhs>
 </emu-production>
 </emu-grammar>
+  <!-- FIXME: Shouldn't this be allowed by the preceding `namespace=annex`? -->
+  <!-- EXPECT_WARNING { "ruleId": "duplicate-definition", "message": "duplicate definition \"SomeAlg\"" } -->
   <emu-annex id="annex11" aoid="SomeAlg">
     <h1><span class="secnum">A.1</span> SomeAlg</h1>
   </emu-annex>
 
+  <!-- FIXME: Shouldn't this be allowed by `namespace=annex2`? -->
```

Comment:
I'm not sure, but I would assume that namespaces apply to the contents of a clause/algorithm, not to the AOID of the algorithm itself.